tests/sqllogictests/suites/base/05_ddl/05_0022_ddl_desc_view.test

```diff
@@ -69,12 +69,11 @@ e DATE NO '1970-01-01' (empty)
 f VARCHAR NO '' (empty)
 
 
-## FLAKY TEST BY dropped table
```

Comment:
Maybe we can add a set session setting in here?

**\"table\" AS table_name,** table is a keyward. so we need add a quote.

statement ok
set sql_dialect='mysql';

Reply:
It's ok now. Let's drop this double quote